common/common.cpp

```diff
@@ -2025,7 +2025,7 @@ void gpt_params_print_usage(int /*argc*/, char ** argv, const gpt_params & param
     options.push_back({ "server",      "       --system-prompt-file FNAME",
                                                                         "set a file to load a system prompt (initial prompt of all slots), this is useful for chat applications" });
     options.push_back({ "server",      "       --log-format {text,json}",
-                                                                        "log output format: json or text (default: json)" });
+                                                                        "log output format: json or text (default: %s)", params.log_json ? "json" : "text" });
```

Comment:
Instead of re-using this arg and risk of changing `server.cpp` code, you can add a new arg for it:

```cpp
options.push_back({ "batched-bench", " --output-format {text,json}", "...." });
```

Add a new variable for it in `common.h`

```cpp
bool batched_bench_output_json = true;
```

Then add handler accordingly:

```cpp
    if (arg == "--log-format") {
        CHECK_ARG
        if (std::strcmp(argv[i], "json") == 0) {
            params.batched_bench_output_json = true;
        } else if (std::strcmp(argv[i], "text") == 0) {
            params.batched_bench_output_json = false;
        } else {
            invalid_param = true;
            return true;
        }
        return true;
    }
```

Reply:
Thanks, but I wonder if that's worth it (given that then this niche option would appear for all of the programs using `gpt_params_parse`).

What if we just do `getenv("LLAMA_BATCHED_BENCH_JSON_OUTPUT")` – would that be better..? There are precedents of using envvars for some things around the repo.

Reply:
For the sake of better documentation and readability, I would recommend having dedicated args when possible, even if it's a niche option. It will help maintainers not to break anything when refactoring the code.

Re. `getenv`, it's currently use exclusively for `llama-server`, so your usage is not expected (c.f. the comment in code):

```cpp
void gpt_params_parse_from_env(gpt_params & params) {
    // we only care about server-related params for now
```

Reply:
> Re. `getenv`, it's currently use exclusively for `llama-server`

TBH, there's e.g. `getenv("LLAMA_CACHE")`, `getenv("LLAMA_TRACE")`, etc. 

Anyway, refactored to a proper `--output-format jsonl` parameter as per your advice.